packages/design-system-v2/package.json

```diff
@@ -1,77 +1,61 @@
 {
   "name": "@appsmithorg/design-system-v2",
-  "version": "1.0.0",
+  "version": "2.0.0-alpha.0",
   "description": "",
-  "module": "build/index.js",
+  "module": "build/index.mjs",
   "types": "build/index.d.ts",
   "files": [
     "build"
   ],
   "scripts": {
-    "build": "rollup -c",
+    "build": "tsc && vite build",
+    "generate:component": "plop component",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
-    "generate:component": "plop component"
+    "build-storybook": "build-storybook"
   },
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@babel/core": "^7.7.4",
-    "@babel/preset-env": "^7.18.2",
-    "@babel/preset-react": "^7.17.12",
-    "@babel/preset-typescript": "^7.17.12",
+    "@babel/core": "^7.20.12",
     "@react-aria/focus": "^3.10.1",
     "@react-aria/radio": "^3.4.2",
     "@react-types/button": "^3.7.0",
-    "@rollup/plugin-commonjs": "^24.0.0",
-    "@rollup/plugin-image": "^3.0.1",
-    "@rollup/plugin-terser": "^0.3.0",
-    "@rollup/plugin-url": "^8.0.1",
+    "@rollup/plugin-typescript": "^11.0.0",
```

Comment:
If we're removing rollup why do we still need the plugin for it?

Reply:
We need this to support typescript in Vite